### PR TITLE
Keep consistent report with x86 for ppc platform.

### PR DESCRIPTION
**Pull request by quanwenli**
1.Remove irq_injs and rpkt_per_irq since it's not supported any
more in kvm state

2.Since powerpc doest not support io_exits in kvm state, uses exits
for all platform.

3.Fail to control last numa node by numactl tool in some ppc machine,
so switch to node 0 instead.

Signed-off-by: Wenli Quan <wquan@redhat.com>

id : 1472058

### CODE REVIEW

**Comment by YanhuiMa** [comment]
If you want to switch to node 0，I think you need to modify netperf.cfg accordingly.
"numa_node = 1"

**Comment by quanwenli** [comment]
param numa_node is used when boot vm, code here is for netperf client which will run on external host, it can not control by param numa_node directly.  but you give me a hint， uses params.get is more good than re.findall. please help check latest update. thanks.

**Comment by YanhuiMa** [comment]
@quanwenli You mean you only need to switch the numa node of netperf to 0, do not need to switch the node of guest to 0? Because sometimes the guest numa pin will also fail on ppc host.

**Comment by WenliQuan** [comment]
Yes, correct.

**Comment by YanhuiMa** [comment]
smp_4.8192m.repeat1.repeat1.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.ppc64le.myprovider.netperf.host_guest: PASS

Acked-by: Yanhui Ma yama@redhat.com

File: generic/tests/netperf.py [hunk 1]
```diff
@@ -534,27 +521,17 @@ def get_state():
...
 
         return state_list
 
     def netperf_thread(i, numa_enable, client_s, timeout):
         cmd = ""
         fname = "/tmp/netperf.%s.nf" % pid
         if numa_enable:
-            output = ssh_cmd(client_s, "numactl --hardware")
-            n = re.findall(r"node (\d+) cpus:", output)[-1]
+            n = abs(int(params.get("numa_node"))) - 1
```
Comment (luckyh) [inline generic/tests/netperf.py]: If you want to use the numa node that you specified, why minus by 1?

Comment (quanwenli) [reply generic/tests/netperf.py]: For keep consistent with virttest/qemu_vm.py

        # Add numa memory cmd to pin guest memory to numa node
        if params.get("numa_node"):
            numa_node = int(params.get("numa_node"))
            if len(utils_misc.get_node_cpus()) < int(params.get("smp", 1)):
                logging.info("Skip pinning, no enough nodes")
            elif numa_node < 0:
                n = utils_misc.NumaNode(numa_node)
                cmd += "numactl -m %s " % n.node_id
            else:
                n = numa_node - 1
                cmd += "numactl -m %s " % n

Comment (quanwenli) [reply generic/tests/netperf.py]: @luckyh , any comments ?

Comment (luckyh) [reply generic/tests/netperf.py]: No more, thanks for your explain.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**